experiments/TestCase/baroclinic_wave.jl

```diff
@@ -316,17 +316,10 @@ function config_diagnostics(FT, driver_config)
         step = FT(1000), # in m
     ))
 
-    #boundaries = [
-    #    FT(-90.0) FT(-180.0) _planet_radius
-    #    FT(90.0) FT(180.0) FT(_planet_radius + info.domain_height)
-    #]
-
-    #resolution = (FT(2), FT(2), FT(1000)) # in (deg, deg, m)
```

Comment:
These comments are kept intentionally. If one does not need the spectra diagnostics group,  then there is no need to compute the lat/long based on the Gaussian points.